CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.1.1] - 2022-04-07
+
+### Added
+
+### Removed
+
+### Changed
+
+### Fixed
+
+- MessagingSession reconnects with refreshed endpoint and credentials if needed
```

Comment:
Could we mention that endpoint URL is deprecated in configuration and we will call `getMessagingSessionEndpoint` automatically ?

Reply:
Also need to update README in the example:
https://github.com/aws/amazon-chime-sdk-js/blob/main/README.md#messaging-session

Reply:
updated thx